conf/test.config

```diff
@@ -19,8 +19,9 @@ params {
     max_time   = '6.h'
 
     // Input data
-    input  = 'https://raw.githubusercontent.com/nf-core/test-datasets/metatdenovo/samplesheet/samplesheet.csv'
+    input    = 'https://raw.githubusercontent.com/nf-core/test-datasets/metatdenovo/samplesheet/samplesheet.csv'
     hmmfiles = 'https://raw.githubusercontent.com/erikrikarddaniel/test-datasets/metatdenovo/testdata/PF00317.hmm,https://raw.githubusercontent.com/erikrikarddaniel/test-datasets/metatdenovo/testdata/PF00848.hmm,https://raw.githubusercontent.com/erikrikarddaniel/test-datasets/metatdenovo/testdata/PF03477.hmm,https://raw.githubusercontent.com/erikrikarddaniel/test-datasets/metatdenovo/testdata/PF13597.hmm'
+    //cat_db   = "://tbb.bio.uu.nl/bastiaan/CAT_prepare/CAT_prepare_20210107.tar.gz"
```

Comment:
This is commented out, do we really want that? Is this test data to make it easier to test cat when needed? If so, fine with me, but I'm not sure it'll pass nf-core reviewers. (We can wait and see with that.) The path seems mangled though.